pkg/pool-weighted/test/IndexPool.test.ts

```diff
@@ -155,12 +155,20 @@ describe('IndexPool', function () {
       pool = await WeightedPool.create(params);
     });
 
+    context('when not called by the owner', () => {
```

Comment:
in the contracts we rename `owner` to `controller` should we do it in the tests as well?

Reply:
Done